osfoffline/polling_osf_manager/osf_query.py

```diff
@@ -242,34 +251,39 @@ def _delete_file_folder(self, remote_file_folder):
         resp.close()
 
     @asyncio.coroutine
-    def make_request(self, url, method=None, params=None, expects=None, get_json=False, timeout=180, data=None):
+    def make_request(self, url, method=None, params=None, expects=None, get_json=False, timeout=180, data=None, headers=None, local_file=None, locks=None):
```

Comment:
This line is too long. better break  it into multi lines. Will make the change when I merge.